src/spyglass/utils/dj_helper_fn.py

```diff
@@ -294,7 +296,7 @@ def fetch_nwb(query_expression, nwb_master, *attrs, **kwargs):
             get_nwb_file(file_path)
 
     query_table = query_expression.join(
-        tbl.proj(nwb2load_filepath=attr_name), log_export=False
```

Comment:
#1192 Has a fix for the log_export for non-SpyglassMixin tables that will still let the export work. Would prefer waiting for that to merge than pushing this change

Reply:
Sounds good, thanks.

Reply:
Can you revert the removals of log_export?

Reply:
done! Hopefully this works.